languages/typescript/src/main/kotlin/io/vrap/codegen/languages/typescript/model/TypeScriptObjectTypeExtensions.kt

```diff
@@ -48,8 +49,8 @@ interface TsObjectTypeExtensions : ExtensionsBase {
     }
 
     private fun relativizePaths(currentModule: String, targetModule: String): String {
-        val currentRelative: Path = Paths.get(currentModule.replace(".", "/"))
-        val targetRelative: Path = Paths.get(targetModule.replace(".", "/"))
+        val currentRelative: Path = Paths.get(currentModule)
```

Comment:
Ok, I thought we already had changed this 😁

Reply:
Yep, turned out i missed it during refactoring 😅